FLIR/conservator/wrappers/collection.py

```diff
@@ -160,56 +161,80 @@ def recursively_get_children(self, include_self=False, fields=None):
             yield collection
             collections.extend(collection.children)
 
-    def get_images(self, fields=None):
+    def get_images(self, fields=None, search_text=""):
         """Returns a query for all images in this collection."""
         images = PaginatedQuery(
-            self._conservator, Image, Query.images, fields=fields, collection_id=self.id
+            self._conservator,
+            Image,
+            Query.images,
+            fields=fields,
+            search_text=search_text,
+            collection_id=self.id,
         )
         return images
 
-    def recursively_get_images(self, fields=None):
+    def recursively_get_images(self, fields=None, search_text=""):
         """Yields all images in this and child collections recursively"""
         for collection in self.recursively_get_children(include_self=True, fields="id"):
-            yield from collection.get_images(fields)
+            yield from collection.get_images(fields, search_text)
 
-    def get_videos(self, fields=None):
+    def get_videos(self, fields=None, search_text=""):
```

Comment:
Nice feature for get_videos() and get_images() to have a search_text parameter :+1:

Reply:
definitely